staging/src/k8s.io/apiserver/pkg/endpoints/handlers/response.go

```diff
@@ -38,8 +39,9 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/handlers/responsewriters"
 	"k8s.io/apiserver/pkg/endpoints/metrics"
 	endpointsrequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/apiserver/pkg/storage"
```

Comment:
Given that we defined the annotations are defined in `staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go`

we should move the function that you're using to staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/helpers.go

Once we do that, we will not need new imports.

Can you please create a separate simple PR that will just do this move?

Reply:
I think you are referring to `storage.HasInitialEventsEndBookmarkAnnotation`. Sure I can do that.

Reply:
eh, it won't work:
```
package k8s.io/apiserver/pkg/endpoints/handlers
	imports k8s.io/apimachinery/pkg/api/equality
	imports k8s.io/apimachinery/pkg/apis/meta/v1
	imports k8s.io/apimachinery/pkg/api/meta
	imports k8s.io/apimachinery/pkg/apis/meta/v1: import cycle not allowed
```

because:
```
func HasInitialEventsEndBookmarkAnnotation(obj runtime.Object) (bool, error) {
	objMeta, err := meta.Accessor(obj)
}
```

unless we would like to change the input object to `obj metav1.Object`